eslint/babel-eslint-config-internal/package.json

```diff
@@ -10,7 +10,7 @@
     "./package.json": "./package.json"
   },
   "peerDependencies": {
-    "@babel/eslint-parser": "^7.10.4",
+    "@babel/eslint-parser": "^7.15.0",
```

Comment:
This is technically a breaking change, but the description of this package is

> The Babel Team's ESLint configuration. Since it's internal, it might not respect semver.

Reply:
It's also marked as private and not published to the registry so that helps